cmake/FindMySql.cmake

```diff
@@ -45,12 +45,6 @@ else()
 	MESSAGE("LIB: ${MYSQL_LIBRARY}")
 endif()
 
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(MYSQL
-  FOUND_VAR MYSQL_FOUND
-  REQUIRED_VARS MYSQL_LIBRARY MYSQL_INCLUDE_DIR
-  )
-
```

Comment:
Any reason you removed that?

Reply:
This failed the cmake a few times, it seems not to be missed at all and it continues to function normally without it.

Reply:
I think it's quite important to handle keywords of find_package() like REQUIRED. 
https://cmake.org/cmake/help/v3.3/module/FindPackageHandleStandardArgs.html

But I will have to test it myself. I'm not entirely sure how this works exactly. My assumption would be that you need that to actually get error messages if mysql is not found. And by removing it, you supressed these errors, which might be important. But that's just a guess.

Reply:
I wait for you to test this better, but for me everything seems to work correctly.